ProductivitySuite/docker_compose/intel/hpu/gaudi/set_env_remote.sh

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Do we need different files to support this feature? I think it's better to integrate them into one file and get environment variables automatically.

Reply:
I created a separate file because in the actual file (to run all services locally) we use a variable called {host_ip} which will point to local machine host ip and different port is configured for different services. But in case of remote host there might be a different endpoints/ports for each service i.e for embedding different endpoint, for llm different endpoint etc.. Because of which I created a separate file with required environment variables so that there is no confusion for users to run it locally or remote. Please let me know if you still feel to integrate it?

Reply:
Thanks for your explanation. I still think it should be integrated into one file, and explain the ${host_ip}/${remote_host_ip} stuff in the `README.md` file. Because for the remote scenario, users should be able to set any service as a remote service, not only tgi/tei embedding.

Further more, please refer to [this pr](https://github.com/opea-project/GenAIExamples/pull/977/files#diff-3c252d2e79c2d8f15044066fb23271b63448d47fc6da0a7410e976c33f40f2e0). The environment variables for docker compose have been replaced with service name, and each examples should follow this schema for users to deploy services easier.

Thank you.

Reply:
Removed set_env_remote.sh and handled it in the README.md file itself.